tools/update_extern.py

```diff
@@ -0,0 +1,169 @@
+#!/bin/env python
+
+"""
+This script is used to update the files in `sunpy/extern` while making new releases.
+"""
+
+import os
+import shutil
+from zipfile import ZipFile
+import requests
+from pathlib import Path
+
+# get parent directory of sunpy
+SUNPY_DIR = Path(__file__).parent.parent
+
+
+# List of packages to update
+PACKAGES = ["appdirs", "distro", "inflect", "parse"]
+
+# Dictionary to store the authors of the packages
+AUTHORS = {
+    "appdirs": "ActiveState",
+    "distro": "python-distro",
+    "inflect": "jaraco",
+    "parse": "r1chardj0n3s",
+}
+
+
+def get_latest_version(package):
+    """
+    Get the latest version of the package.
+    """
+    url = f"https://api.github.com/repos/{AUTHORS[package]}/{package}/releases/latest"
```

Comment:
What about using tags instead?

Reply:
I am not sure what you mean, but this function returns the latest `version tag` of the package.
```py
return response.json()["tag_name"]
```
For example, using this endpoint: https://api.github.com/repos/sunpy/sunpy/releases/latest we can get `"tag_name": "v4.0.0"`

Reply:
Sounds good.